packages/yuudachi/locales/en-US/translation.json

```diff
@@ -459,5 +456,89 @@
 				"description": "• Moderator: {{- moderator}}\n• Hit count: {{count}}"
 			}
 		}
+	},
+	"formatters": {
+		"formatting": {
+			"guildmember": "{{-id}}| Joined: {{-join}} | Created: {{-creation}} | {{-tag}}"
```

Comment:
```suggestion
			"guild_member": "{{- id}} | Joined: {{- join}} | Created: {{- creation}} | {{- tag}}"
```

Also make sure to update all instances that used `guildmember` to `guild_member`

Reply:
`{{- id}} |`
          ^
The removal of the space here is intentional and will be fine bc of the padding

Reply:
Fixed in 731a286